project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.5.8
+sbt.version=1.6.2
```

Comment:
```suggestion
sbt.version=1.5.8
```
Please do not change this as part of this PR

Reply:
Accidentally let this slip through, will revert on next push